src/components/FilterTopGrower.js

```diff
@@ -166,6 +169,16 @@ function FilterTopGrower(props) {
                 onChange={(e) => setEmail(e.target.value)}
                 onKeyDown={handleEnterPress}
               />
+              <TextField
```

Comment:
Can we move this one place up to be after Last Name?
That seems like a more logical location.